### PR TITLE
Class refactoring

### PR DESCRIPTION
**Pull request by sangwa7**
### Owesome Book

- Added classes
- Added constructors.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,7 +1,78 @@
...
+table {
+  background-color: aliceblue;
+  width: 60%;
+  margin-left: 10%;
+}
+
+.books {
+  background-color: aqua;
+  width: 100%;
+}
```
Comment (prolajumokeoni) [inline css/style.css:54]: - [ ] Kindly make your design match the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png) 👍🏿

**Review by dasileker — APPROVED** [review]
### **Hi @sangwa7,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -53,10 +53,16 @@ set up on github pages
...
+👤 **Sangwa Didier**
+
 - GitHub: [@EzekielUtshudi](https://github.com/EzekielUtshudi)
 - Twitter: [@UtshudiEzekiel](https://twitter.com/UtshudiEzekiel)
 - LinkedIn: [@UtshudiEzekiel](https://www.linkedin.com/in/ezekiel-utshudi-195782162/)
 
+- GitHub: [@SangwaDidier](https://github.com/sangwa7)
+- Twitter: [@DidierSangwa](https://twitter.com/Didiersangwa)
+- LinkedIn: [@DidierSangwa](https://www.linkedin.com/in/Sanwa-didier/)
+
```
Comment (prolajumokeoni) [inline README.md:65]: - [ ] Kindly add name of author  to the second author's  social media links, check example below ⬇️ 
```suggestion
👤 **Sangwa Didier**
- GitHub: [@SangwaDidier](https://github.com/sangwa7)
- Twitter: [@DidierSangwa](https://twitter.com/Didiersangwa)
- LinkedIn: [@DidierSangwa](https://www.linkedin.com/in/Sanwa-didier/)

```

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @sangwa7 and @EzekielUtshudi 👋🏿 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Github flow correctly used ✔️ 

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**